parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -42,20 +42,20 @@ installContainerRuntime() {
     if [[ "${NEEDS_CONTAINERD}" == "true" ]]; then
         echo "in installContainerRuntime - KUBERNETES_VERSION = ${KUBERNETES_VERSION}"
         local containerd_version
-        if [[ -f "$COMPONENTS_FILEPATH" ]]; then
+        if [[ ! -f "$COMPONENTS_FILEPATH" ]]; then
```

Comment:
This should be the bug for the E2E test failure.
Added 1 unit test to cover this too.